src/main/java/seedu/taassist/logic/commands/actions/UiAction.java

```diff
@@ -0,0 +1,12 @@
+package seedu.taassist.logic.commands.actions;
+
+/**
+ * Represents the different actions the UI can perform.
+ */
+public enum UiAction {
```

Comment:
Consider this class structure: 
```
actions
  |-- storageaction
  |    |-- StorageAction [abstract, has act(Storage)]
  |    |-- ExportCsvStorageAction
  |    |-- exceptions
  |          |-- StorageActionException
  |-- uiaction
        |-- UiAction [abstract, has act(MainWindow)]
        |-- UiHelp (singleton)
        |-- UiFocus (singleton)
        |-- ...
```
However, I don't like the `UiHelp#act` taking in a `MainWindow`. Can you think of a better solution?

Reply:
Not really. Unless we are planning to dive much deeper and reconstruct the structure for UI logic, we should keep it simple like this for now.